src/sympc/store/crypto_primitive_provider.py

```diff
@@ -86,3 +93,31 @@ def get_state() -> str:
         """
         res = f"Providers: {list(CryptoPrimitiveProvider._func_providers.keys())}\n"
         return res
+
+    @staticmethod
+    def start_logging(generate_file: bool = False) -> None:
```

Comment:
MAYBE: Move the ```generate_file``` arg to the ```stop_logging```. In that way, we can remove it from class.

Reply:
done